package.json

```diff
@@ -6,6 +6,9 @@
   "homepage": "https://hypothes.is",
   "bugs": "https://github.com/hypothesis/client/issues",
   "repository": "hypothesis/client",
+  "dependencies": {
+    "@hypothesis/frontend-shared": "1.0.0"
```

Comment:
This should be listed in `devDependencies` rather than `dependencies`, since the code from `frontend-shared` that is used by the application will end up being included in the bundle.

Reply:
Right, this makes sense, we're bundling everything at build time.